### PR TITLE
6229 add additional details

### PR DESCRIPTION
**Pull request by bthaile**
reuse Market-properties component to not-show or show 'Additional Details' to give user ability show/hide details in the reporting-report component. Wasn't able to keep it totally self contained based on current Mock layouts.

https://app.clubhouse.io/augur/story/6229/reporting-form-add-market-card-and-additional-details

### CODE REVIEW

**Comment by justinbarry** [comment]
@bthaile The mockups looks like the text is constrained horizontally to be inline with the market preview content. See:

screen shot 2018-02-26 at 1 24 02 pm

File: src/modules/reporting/components/reporting-report/reporting-report.jsx [hunk 1]
```diff
@@ -103,8 +110,26 @@ export default class ReportingReport extends Component {
             history={p.history}
             cardStyle="single-card"
             buttonText="View"
+            showAdditionalDetailsToggle={s.showToggleDetails}
```
Comment (justinbarry) [inline src/modules/reporting/components/reporting-report/reporting-report.jsx]: Could we move this out of the MarketPreview component and style it to look like it belongs there?

File: src/modules/reporting/components/reporting-report/reporting-report.jsx [hunk 2]
```diff
@@ -103,8 +110,26 @@ export default class ReportingReport extends Component {
...
+          <div className={Styles[`ReportingReportMarket__details-container-wrapper`]}>
+            <div className={Styles[`ReportingReportMarket__details-container`]}>
+              <div className={Styles.ReportingReportMarket__details}>
+                <span>
+                  {p.market.extraInfo}
+                </span>
+              </div>
+              <div className={Styles[`ReportingReportMarket__resolution-source`]}>
+                <h4>Resolution Source:</h4>
+                <span>{p.market.resolutionSource || 'Outcome will be determined by news media'}</span>
```
Comment (justinbarry) [inline src/modules/reporting/components/reporting-report/reporting-report.jsx]: Should this be hard coded?

File: src/modules/reporting/components/reporting-report/reporting-report.styles.less [hunk 3]
```diff
@@ -4,3 +4,40 @@
   color: @color-lightgray;
   margin-top: 5rem;
 }
+
+.ReportingReportMarket__details {
```
Comment (justinbarry) [inline src/modules/reporting/components/reporting-report/reporting-report.styles.less]: I would consider moving this out to another BEM namespace. Not a deal breaker but could help with encapsulating styling.

**Comment by justinbarry** [comment]
@sharkcrayon Do we want a css animation of some kind when toggling?

File: src/modules/reporting/components/reporting-report/reporting-report.jsx (hunk 1 above)
Comment (bthaile) [reply src/modules/reporting/components/reporting-report/reporting-report.jsx]: Definitely missed this, will correct.

File: src/modules/reporting/components/reporting-report/reporting-report.jsx (hunk 2 above)
Comment (bthaile) [reply src/modules/reporting/components/reporting-report/reporting-report.jsx]: Yes, there are only 2 options when creating a market. The market creator can fill in their website url or say the source will be determined by news media.

File: src/modules/reporting/components/reporting-report/reporting-report.styles.less (hunk 3 above)
Comment (bthaile) [reply src/modules/reporting/components/reporting-report/reporting-report.styles.less]: The styling is for content in the Reporting Report component so I'd like the keep the styling in this same file.

File: src/modules/reporting/components/reporting-report/reporting-report.jsx (hunk 1 above)
Comment (bthaile) [reply src/modules/reporting/components/reporting-report/reporting-report.jsx]: Based on the Mocks this is the best component architecture I could come up with. Hopefully the Mocks will change to allow for a better component layout. If anyone knows of a better way to architect this I'll need direction.

**Comment by justinbarry** [comment]
Looks good!

screen shot 2018-02-27 at 1 45 25 am

**Pull request merged**